HLTrigger/Configuration/python/customizeHLTforAlpaka.py

```diff
@@ -1090,6 +1090,172 @@ def customizeHLTforAlpakaEcalLocalReco(process):
+    process.hltTowerMakerForAllSerialSync.hbheInput = "hltHbherecoSerialSync"
+    process.hltAK4CaloJetsIDPassedSerialSync.JetIDParams.hbheRecHitsColl = "hltHbherecoSerialSync"
+    process.hltHbheRecHitSoASerialSync.src = "hltHbherecoSerialSync"
+    process.hltMuonsSerialSync.TrackAssociatorParameters.HBHERecHitCollectionLabel = "hltHbherecoSerialSync"
+    process.hltMuonsSerialSync.CaloExtractorPSet.TrackAssociatorParameters.HBHERecHitCollectionLabel = "hltHbherecoSerialSync"
+    process.hltMuonsSerialSync.JetExtractorPSet.TrackAssociatorParameters.HBHERecHitCollectionLabel = "hltHbherecoSerialSync"
+
+    # run the HCAL local reconstruction, potentially offloading the MHI step to the device
+    process.HLTDoLocalHcalSequence = cms.Sequence(
+      process.hltHcalDigis +
+      process.hltHcalDigisSoA +
+      process.hltHbheRecoSoA +
+      process.hltHbhereco +
+      process.hltHfprereco +
+      process.hltHfreco +
+      process.hltHoreco )
+
+    # run the HCAL local reconstruction on the host
+    process.HLTDoLocalHcalSequenceSerialSync = cms.Sequence(
+      process.hltHcalDigis +
+      process.hltHcalDigisSoASerialSync +
+      process.hltHbheRecoSoASerialSync +
+      process.hltHbherecoSerialSync +
+      process.hltHfprereco +
+      process.hltHfreco +
+      process.hltHoreco )
+
+    # run the HBHE local reconstruction, potentially offloading the MHI step to the device
+    process.HLTStoppedHSCPLocalHcalReco = cms.Sequence(
+      process.hltHcalDigis +
+      process.hltHcalDigisSoA +
+      process.hltHbheRecoSoA +
+      process.hltHbhereco )
+
+    # PFJet reconstruction running on the host
+    process.HLTPFHcalClusteringSerialSync = cms.Sequence(
+      process.hltHbheRecHitSoASerialSync +
+      process.hltParticleFlowRecHitHBHESoASerialSync +
+      process.hltParticleFlowRecHitHBHESerialSync +
+      process.hltParticleFlowClusterHBHESoASerialSync +
+      process.hltParticleFlowClusterHBHESerialSync +
+      process.hltParticleFlowClusterHCALSerialSync )
+
+    paths_AlCa_PFJet40_CPUOnly_v = [ path for path in process.paths if path.startswith('AlCa_PFJet40_CPUOnly_v') ]
+    if paths_AlCa_PFJet40_CPUOnly_v:
+      pathname = paths_AlCa_PFJet40_CPUOnly_v[0]
+      setattr(process, pathname, cms.Path(
+        process.HLTBeginSequence +
+        process.hltL1sZeroBias +
+        process.hltPreAlCaPFJet40CPUOnly +
+        process.HLTDoLocalHcalSequenceSerialSync +
+        process.HLTAK4CaloJetsSequenceSerialSync +
+        process.hltSingleCaloJet10SerialSync +
+        process.HLTAK4PFJetsSequenceSerialSync +
+        process.hltPFJetsCorrectedMatchedToCaloJets10SerialSync +
+        process.hltSinglePFJet40SerialSync +
+        process.HLTEndSequence )
+      )
+
+    # compare the HCAL local reconstruction running on the device and on the host
+    paths_DQM_HcalReconstruction_v = [ path for path in process.paths if path.startswith('DQM_HcalReconstruction_v') ]
```

Comment:
@mmusich I'm not familiar with how the customize function works. Would these changes be sufficient? 
```suggestion
+    paths_DQM_HcalReconstruction_v = [ path for path in process.paths if path.startswith('DQM_HcalReconstruction_v') or path.startswith("DQM_HIHcalReconstruction_v") ]
```

Reply:
@kakwok  this is not sufficient, what you want is:
```python
    for pathNameMatch in ['DQM_HcalReconstruction_v', 'DQM_HIHcalReconstruction_v']:
        dqmHcalRecoPathName = None
        for pathName in process.paths_():
            if pathName.startswith(pathNameMatch):
                dqmHcalRecoPath = getattr(process, pathName)
                dqmHcalRecoPath.insert(dqmHcalRecoPath.index(process.HLTPFHcalClustering), getattr(process, 'HLTDoLocalHcalSequenceSerialSync'))
                for delmod in ['hltHcalConsumerCPU', 'hltHcalConsumerGPU']:
                    if hasattr(process, delmod):
                        process.__delattr__(delmod)
```